src/assets/styles/custom-scroll.less

```diff
@@ -4,6 +4,8 @@
 
 .side-scroll {
   padding-bottom: 0px;
+  border-top: 1px solid @dark-background;
+  transition: border 0.7s;
```

Comment:
There's also a specific bit of LESS in search.less for something similar:

  ```.sidetab > div:first-of-type search .search-result {
    border-top: @search-splitter-style;
  }```

Can something be refactored to make these more common?